iosHackthonApp/Modules/Shared/Strings/Sources/Strings/Strings.swift

```diff
@@ -33,6 +33,22 @@ public struct Strings { // TODO: these can all be public enums, plus public exte
         public static let signupButtonTitle = "Don't have an account? Sign up"
     }
 
+	public struct SignUp {
+		public static let welcomeScreenTitle = "Hello!"
+		public static let welcomeScreenInfo = "Community Kitchen helps provide free food to those who need it by enabling the community to share extra or unwanted food"
+		public static let welcomeButtonTitle = "Register with email"
+		public static let signUpWithEmailTitle = "Email"
+		public static let signUpWithEmailPlaceHolder = "Enter your email address"
+		public static let signUpWithEmailInfo = "First we'll need to know the email address you want to sign up with.\n \nWe will never share your email with any 3rd parties or send you any emails you dont request."
+		public static let signUpWithEmailButtonTitle = "Continue to password"
+		public static let signUpWithPasswordTitle = "Password"
+		public static let signUpWithPasswordPlaceHolder = "Enter a password"
+		public static let signUpWithPasswordInfo = "Next please choose a strong password.\n \nWe recommend using a strong password generated by password manager"
```

Comment:
Can we change this line to:
`Next, please choose a strong password.\n \nWe recommend using a strong password generated by a password manager.`

Reply:
Done